devicetypes/smartthings/zwave-switch.src/zwave-switch.groovy

```diff
@@ -156,13 +159,34 @@ def poll() {
 	])
 }
 
+
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
+	refresh()
+}
+
 def refresh() {
 	delayBetween([
 		zwave.switchBinaryV1.switchBinaryGet().format(),
 		zwave.manufacturerSpecificV1.manufacturerSpecificGet().format()
 	])
 }
 
+def healthPoll() {
+	def cmds = refresh()
+	cmds.each{ sendHubCommand(new physicalgraph.device.HubAction(it))}
+}
+
+def configure() {
+	unschedule()
+	schedule("0 0/5 * * * ? *", "healthPoll")
```

Comment:
The DTH scheduling method runEvery5Minutes() should be going out to data-mgmt production this Wed. 9/29. Let's test this method in dev now and have one PR to merge rather than 2. 
https://smartthings.atlassian.net/wiki/display/DATAMGMT/2016/09/26/2016-09-29+Release